### PR TITLE
Fix failures

### PR DESCRIPTION
**Pull request by abuzar-deriv**
1. Add scrollIntoView to `c_closeNotificationHeader`
2. Added cy.c_completeTradersHubTour() to tests
3. Updated `c_switchToReal` and `c_switchToDemo` to use testing library locators
4. Updated generateEpoch import path for some tests
5. Added additional check to handle declaration for spain (card ref: https://app.clickup.com/t/20696747/TRAH-2864)

Test Results: 6 failures left, platformNavOptandMult.cy.js is expected to fail due to Smart Trader not loading in staging. Another issue is in picking signup email. Will fix it here: https://app.clickup.com/t/20696747/FEQ-2033. 

image

### CODE REVIEW

File: cypress/e2e/wip/Tradershub/platformNavOptandMult.cy.js [hunk 1]
```diff
@@ -4,8 +4,8 @@ describe('QATEST-5948: Verify platforms navigations on Options and Multipliers',
   it('Should navigate to correct platform on clicking Open button', () => {
     cy.c_login()
     cy.c_visitResponsive('/appstore/traders-hub', 'large')
-    const derivAppProdUrl = Cypress.env('prodURL')
-    const derivAppStagingUrl = Cypress.env('stagingUrl')
+    const derivAppProdUrl = `${Cypress.env('prodURL')}?chart_type=area&interval=1t&symbol=1HZ100V&trade_type=accumulator`
```
Comment (markodonnell-deriv) [inline cypress/e2e/wip/Tradershub/platformNavOptandMult.cy.js:7]: Are we OK to hard code these symbols?

**Review by markodonnell-deriv — COMMENTED** [review]
Looks good, just one minor comment.

**Review by bilalmajid-deriv — APPROVED** [review]

File: cypress/e2e/wip/Tradershub/platformNavOptandMult.cy.js (hunk 1 above)
Comment (bilalmajid-deriv) [reply cypress/e2e/wip/Tradershub/platformNavOptandMult.cy.js:7]: @abuzar-deriv why not use a regex over here?

Comment (abuzar-deriv) [reply cypress/e2e/wip/Tradershub/platformNavOptandMult.cy.js]: Updated to regex. 
Test: 
image

**Review by markodonnell-deriv — APPROVED** [review]
Looks good

**Comment by abuzar-deriv** [comment]
Tests are already fixed by @indulekha-deriv in PR https://github.com/deriv-com/e2e-deriv-app/pull/178. I still need some of these updates but I'll push them in a new PR. Closing this one.

**Pull request closed without merging**